.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,75 @@
+name: Docker
+
+on:
+  push:
+    # Publish `main` as Docker `latest` image.
+    branches:
+      - main
+
+    # Publish `v1.2.3` tags as releases.
+    tags:
+      - v*
+
+  # Run tests for any PRs.
+  pull_request:
+
+env:
+  IMAGE_NAME: image
```

Comment:
Anyone have any thoughts on what we want to call the image name?

docker.pkg.github.com/civictechindex/cti-website-backend/`image`:latest

Reply:
Set this image name to service.